lang_qc/models/pacbio/experiment.py

```diff
@@ -93,6 +93,13 @@ class PacBioExperiment(BaseModel):
         unlikely case of multiple library types.
         """,
     )
+    library_tube_barcode: str = Field(
```

Comment:
I think we should rename this to `pool_name` Something that claims to be a barcode, but is clearly not a barcode creates confusion.

Reply:
@ces Do you have any good ideas?